core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -119,6 +119,7 @@ public class TabletMetadata {
   private TabletHostingGoal goal = TabletHostingGoal.ONDEMAND;
   private boolean onDemandHostingRequested = false;
   private TabletOperationId operationId;
+  private Map<Long,TServerInstance> refreshIds;
```

Comment:
is this still being used?

Reply:
no, removed in [27fc328](https://github.com/apache/accumulo/pull/3350/commits/27fc328d10979710023da6207f9eba92250dca3b)